src/run.cpp

```diff
@@ -25,6 +25,10 @@
 #include <unistd.h>
 #include <signal.h>
 
+#if defined(__FreeBSD__)
+#include <sys/wait.h>
```

Comment:
I'd just include it unconditionally - doesn't do any harm and easier to parse. Out of interest what was undefined?